week1/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>help hamsick</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="headerColorBox">
+      <h1>💛🍔 햄식이를 도와줘 🍔💛</h1>
+      </div>
+    </header>
+
+    <main>
+      <aside class="sidebar">
+        <section class="option_card">
+          <h2 class="optionTitle">다른거 먹자</h2>
+          <div class="option"> 
+            <p class="macdonalds">맥도날드</p>
+            <p class="burgerking">버거킹</p>
+          </div>
+        </section>
+      </aside>
+      <section class="burger">
+        <article class="burger_card">
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1583727855319.png" alt="빅맥사진">
+          <div class="burger_content">
+            <h2 class="buger_name">빅맥</h2>
```

Comment:
heading 테그는 웹페이지 전체에서 대표할 수 있는 내용에 쓰여서 여기선 strong테그가 더 적절한 것 같아요!
참고자료 ) 
https://blog.hubspot.com/marketing/header-tags
```suggestion
            <strong class="buger_name">빅맥</strong>
```

Reply:
앗 질문이 있어요 커멘트 써주신거 기반으로 서칭을 조금해봤는데 strong이랑 비슷하게 강조 처리를 해주는 태그가 em , b도 있더라구요! 그러면 이 과제에서는 strong이 가장 적절할까요?

Reply:
@songah119  b테그같은 경우 "중요하다"는 의미를 담기보다는 단순히 글자를 굵게 만들어주는 느낌이고, em테그와 strong테그의 경우엔 "중요하다"는 의미를 담고 있는 느낌이에요! em테그보다 strong테그가 더 중요도가 높은 느낌입니다!

근데 제가 송아님 리뷰한 이후 과제를 하면서 좀 더 찾아봤는데, article테그 내부에는 heading테그가 있는 게 권장되고 있더라고요,, 헤헤 ,,, 그래서 여기서 층위에 맞게 h3테그를 쓰는 편이 좋을 것 같다는 생각이.. 이제야 들었네여 ㅎ...ㅎ.. 죄송합미다 😅
```suggestion
            <h3 class="buger_name">빅맥</h3>
```